src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -0,0 +1,86 @@
+import { Toolbar } from "../../../../../components/client/toolbar/Toolbar";
+import { DashboardTopBanner } from "./DashboardTopBanner";
+import { useL10n } from "../../../../../hooks/l10n";
+import type { UserBreaches } from "../../../../../functions/server/getUserBreaches";
+import { ExposureCard } from "../../../../../components/client/ExposureCard";
+import { ExposuresFilter } from "../../../../../components/client/ExposuresFilter";
+
+export type Props = {
+  user: Session["user"];
+  userBreaches: UserBreaches;
+  locale: string;
+};
+
+export const View = (props: Props) => {
+  const l10n = useL10n();
+
+  return (
+    <div className={styles.wrapper}>
+      <Toolbar user={props.user}>
+        TODO:{" "}
+        <a href="https://react-spectrum.adobe.com/react-aria/useTabList.html">
+          add a tab list
+        </a>
+      </Toolbar>
+      <div className={styles.dashboardContent}>
+        <DashboardTopBanner type={"LetsFixDataContent"} chart={<></>} />
+        <section className={styles.exposuresArea}>
+          <h2 className={styles.exposuresAreaHeadline}>
+            {l10n.getString("dashboard-exposures-area-headline")}
+          </h2>
+          <p className={styles.exposuresAreaDescription}>
+            {l10n.getString("dashboard-exposures-area-description", {
+              // TODO: Use real user data
+              exposures_total_num: 1337,
+              // TODO: Should this be emailTotalCount or emailVerifiedCount?
+              data_breach_total_num: props.userBreaches.emailVerifiedCount,
+              data_broker_total_num: 1337,
+            })}
+          </p>
+          <div className={styles.exposuresFilterWrapper}>
+            <ExposuresFilter />
+          </div>
+          <ul className={styles.exposureList}>
+            {props.userBreaches.breachesData.verifiedEmails
+              .map((verifiedEmail) => {
+                const breachCardsForThisEmail = verifiedEmail.breaches.map(
+                  (breach) => {
+                    return (
+                      <li
+                        key={`${verifiedEmail.email}_${breach.Id.toString()}`}
+                        className={styles.exposureListItem}
+                      >
+                        <ExposureCard
+                          exposureImg={TwitterImage}
+                          exposureData={breach}
+                          exposureName={breach.Name}
+                          exposureDetailsLink={""}
+                          dateFound={breach.AddedDate}
+                          statusPillType={"fixed"}
+                          locale={props.locale}
+                        />
+                      </li>
+                    );
+                  }
+                );
+                // Technically a JSX.Element can be `any`, but we know it's not.
+                // (At least, I *think* that's why this rule triggers.)
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Question: I’m not sure that disabling this rule es needed — not seeing a TS error on my end. Is this still happening for you?